index.html

```diff
@@ -754,13 +754,13 @@ <h3>Validate Algorithm</h3>
       </ol>
 
       <p>
-When dereferencing a `statusListCredential` URL, implementations MAY provide a
-mechanism to query the status list at a particular point in time. If such a
+When a `statusListCredential` URL is dereferenced, server implementations MAY provide a
+mechanism to dereference the status list as of a particular point in time. If such a
 feature is supported, and if query parameters are supported by the URL scheme,
 then the name of the query parameter MUST be `timestamp` and the value MUST be
 a valid URL-encoded [[XMLSCHEMA11-2]] dateTimeStamp string value. The resulting
 status list credential SHOULD be the status list as it existed at the given
```

Comment:
Possibly this should have its own specific error, as have recently been created for other specific conditions.

Note that this change should replace @David-Chadwick's [first suggestion](https://github.com/w3c/vc-bitstring-status-list/pull/112/files/e02c6e976ccf5d10d0224f41ae0506e4cd1e83be..1613363d0221ec44727c3226b05321d27049ecfa#r1438636997) and be accompanied by his [second](https://github.com/w3c/vc-bitstring-status-list/pull/112/files/e02c6e976ccf5d10d0224f41ae0506e4cd1e83be..1613363d0221ec44727c3226b05321d27049ecfa#r1438637110).
```suggestion
a valid URL-encoded [[XMLSCHEMA11-2]] dateTimeStamp string value. The result of
dereferencing such a timestamped URL MUST be a status list credential containing
either the status list as it existed at the given point in time, or an error.
```